src/macros.jl

```diff
@@ -728,18 +758,19 @@ write
 ##############################################################################
 
 function subset_helper(x, args...)
-    exprs, outer_flags = create_args_vector(args...)
+    x, exprs, outer_flags, kw = get_df_args_kwargs(x, args...; wrap_byrow = false)
+
     t = (fun_to_vec(ex; no_dest=true, outer_flags=outer_flags) for ex in exprs)
     quote
-        $subset($x, $(t...); skipmissing=true)
+        $subset($x, $(t...); (skipmissing = true,)..., $(kw...))
```

Comment:
is passing `skipmissing` explicitly tested? Why does `skipmissing` need to be in a tuple and later `...`?

Reply:
It is explicitely tested for throwing an error. Every keyword argument is tested. 

I do not know why exactly I need a tuple and splatting, but by omitting that I get 

```
julia> @subset(df, :x .== 1; skipmissing = false)
ERROR: syntax: keyword argument "skipmissing" repeated in call to "DataFrames.subset" around /home/peterwd/Documents/Development/DataFramesMeta/src/macros.jl:765
Stacktrace:
 [1] top-level scope
   @ REPL[8]:1
```

so it seems necessary.

Reply:
ah - OK. So that is why it is needed. If you pass it explicitly uniqueness is tested, if you pass collections they are merged. This is strange, as this is not required in other contexts:
```
julia> (;a=1, [:a => 2]...)
(a = 2,)
```
(and you get merging without having to wrap the first in a tuple.